Pokemon Player/src/Battle.cpp

```diff
@@ -33,33 +42,38 @@ void Battle::fight()
 	{
 		//Read the current state of the battle
 		readRam();
+
+		//Update bost lead's stats
+		updatePoke(0);
+		updatePoke(-1);
```

Comment:
This line would be more clear if you used a constant for -1 called OPPONENT_INDEX or something.